File: 인범/2026_06_04/TreasureMap.java
class Solution {
    public String[] solution(int n, int[] arr1, int[] arr2) {
        String[] answer = new String[n];
        
        for (int i =0; i<n; i++){
            String Temp = "";
            int num = arr1[i] | arr2[i];
            for (int j=0; j<n; j++){
                int k = 1<<j;
                if ((num & k) == k) {
                    Temp = "#" + Temp;
                }
                else {
                    Temp = " " + Temp;
                }
            }
            answer[i]= Temp;
        }
        
        return answer;
    }
}

File: 인범/2026_06_04/treasure_map.py
def solution(n, arr1, arr2):
    compositions=[]
    for n1,n2 in zip(arr1,arr2):
        temp = ""
        for i in range(0,n):
            k = 1<<i
            if k & (int(n1) | int(n2)) : temp = "#"+temp
            else : temp = " "+temp
        compositions.append(temp)
        
    return compositions